openreview/conference/templates/reviewerWebfield.js

```diff
@@ -102,20 +102,23 @@ var buildNoteMap = function(noteNumbers) {
 
 // AJAX functions
 var getReviewerNoteNumbers = function() {
-  return Webfield.get('/groups', {
+  return Webfield.getAll('/groups', {
     regex: ANONREVIEWER_WILDCARD,
     member: user.id
-  }).then(function(result) {
-    if (!result.groups) {
-      return [];
-    }
-    return result.groups.reduce(function(groupByNumber, group) {
-      var number = getNumberFromGroup(group.id, 'Paper');
-      if (number) {
-        groupByNumber[number] = group.id;
-      }
-      return groupByNumber;
-    }, {});
+  }).then(function(groups) {
+
+    var anonGroups = _.filter(groups, function(g) { return g.id.includes('Reviewer_'); });
```

Comment:
for old webfields with using reviewer1, reviewer2, reviewer3
this may need to be changed to g.id.includes('AnonReviewer')

refer to reviewer console of ICLR2021 which has been fixed